packages/example/cypress/integration/examples/network_requests.spec.js

```diff
@@ -87,6 +87,60 @@ context('Network Requests', () => {
     })
   })
 
+  it('cy.request() - pass result to the second request', () => {
```

Comment:
updated kitchensink example

Reply:
curious why this is part of this PR

Reply:
because when I did `npm i` branching of the latest `develop` it actually built and copied new kitchensink example spec files. 
We should update the kitchensink files whenever we bump the dependency in `package.json`